cli/cook/subcommands/jobs.py

```diff
@@ -158,7 +185,10 @@ def register(add_parser, add_defaults):
                                              "alphanumeric characters, '.', '-', '_', and '*' as a wildcard)")
     parser.add_argument('--limit', '-l', help=f'limit the number of results (default = {DEFAULT_LIMIT})',
                         type=check_positive)
-    parser.add_argument('--json', help='show the data in JSON format', dest='json', action='store_true')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--json', help='show the data in JSON format', dest='json', action='store_true')
+    group.add_argument('-1', help='list one job per line, without table formatting',
```

Comment:
Weren't we going to have a long form of this option too? Maybe `--entity-urls`?

Reply:
I added a long form: `--urls`